cluster-autoscaler/cloudprovider/huaweicloud/huaweicloud_cloud_provider.go

```diff
@@ -185,7 +185,6 @@ func getAutoscaleNodePools(manager *huaweicloudCloudManager, opts config.Autosca
 	}
 
 	clusterUpdateLock := sync.Mutex{}
-	deleteMux := sync.Mutex{}
 
 	// Given our current implementation just support single node pool,
```

Comment:
No problem here, but need to point out that the `clusterUpdateLock` and `deleteMux` should be defined in the loop so that we can ensure the same mutex can't be shared with different node groups.

Reply:
Yeah you're right. I'll move the clusterUpdateLock inside the for loop below. Thank you for the comment!